### DISCUSSION TITLE
[Thesis] Quizfragen importieren: QTI nach DSL

### DISCUSSION THREAD

**Discussion by cagix** [Programmiermethoden/Dungeon#1379] [Ideas]
ILIAS kann fertige Aufgaben im [QTI](https://en.wikipedia.org/wiki/QTI)-Format exportieren. Es wäre hilfreich, diese automatisiert in unser Framework importieren zu können. Vermutlich wäre es sinnvoll, die eingelesenen Fragen vom QTI-Format in unsere DSL zu übersetzen und dann das vorhandene/geplante Tooling arbeiten zu lassen.

https://digitaliser.getmarked.ai/blog/complete-guide-to-qti/

**Comment by Ironeer** [comment]
Als ich recherchiert habe, ob den ILIAS QTI Export in die DSL umzuwandeln mein Forschungsprojekt werden könnte, bin ich auf folgende eher abschreckende Informationen über die ILIAS QTI-Implementierung gestoßen, die ich hier dokumentieren möchte:
* [Das ILIAS Feature Dokument 'Allow QTI import and export of tests' wurde von ILIAS mit dem Kommentar 'We close this as part of the cleaning up effort undertaken' geschlossen](https://docu.ilias.de/ilias.php?baseClass=ilwikihandlergui&cmdNode=16n:r6&cmdClass=ilobjwikigui&cmd=viewPage&ref_id=1357&page=Allow_QTI_import_and_export_of_tests)
* [Zitat aus einer ILIAS Community Diskussion 'Die QTI-Implementation ist eine ziemlich alte Implementation'](https://docu.ilias.de/ilias.php?baseClass=ilRepositoryGUI&cmd=viewThread&cmdClass=ilObjForumGUI&ref_id=1881&thr_pk=5404)
* [Unbeantwortete Frage, welche QTI Version ILIAS untertützt](https://docu.ilias.de/ilias.php?baseClass=ilRepositoryGUI&cmd=viewThread&cmdClass=ilObjForumGUI&ref_id=1881&thr_pk=1045)

**Reply by cagix** [reply]
danke dir!
